server/src/lib/data/request-handlers/get-user-id-from-session-or-bail.ts

```diff
@@ -0,0 +1,12 @@
+import type { Request, Response } from "express";
+
+export const getUserIdFromSessionOrBail = (req: Request, res: Response) => {
```

Comment:
Maybe we should leave the "bail" part of this to the call site, because, since this thing isn't actually middleware, the typing isn't robust enough to know not to pass on to the rest of the logic, so when we forget to wrap, at the call site, the query handler with an `if (user_id) ...`, we could get a "cannot send headers after being sent ..." error.